From 36a6f6977152958d0045447cc9cbb1fe737a5402 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentnl@gentoo.org>
Date: Mon, 16 Mar 2020 08:50:04 +1300
Subject: futures: tests/try_join.rs: Don't break w/o features

Only really appropriate action here is to gate the whole file with
`#![cfg()]`

Tests now build and pass with all valid feature combinations

See previous commits for details and rationale
---
 tests/try_join.rs | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/try_join.rs b/tests/try_join.rs
index 6c6d084..0861c1e 100644
--- a/tests/try_join.rs
+++ b/tests/try_join.rs
@@ -1,3 +1,5 @@
+#![cfg(feature = "executor")] // executor::
+#![cfg(feature = "async-await")] // try_join!
 #![deny(unreachable_code)]
 
 use futures::{try_join, executor::block_on};
-- 
2.25.0

